dts/bindings/misc/nxp,rdc-policy.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+  rdc:
+    type: int
+    description: |
+      Set the RDC permission for specified peripheral: Resource Domain Controller (RDC)
+      supports flexible configuration on IP’s access permission, each individual IP can,
+      for example, be configured as A-core only or M-core only. This property uses some
+      macros defined in imx_rdc.h to configure current peripheral's access permission.
+      For example to allow both A53 and M7 Core to have read/write permission to current
+      peripheral:
+      rdc = <(RDC_DOMAIN_PERM(A53_DOMAIN_ID, RDC_DOMAIN_PERM_RW)|\
```

Comment:
what I would like to see is all the options of the macros listed in this description

Reply:
I don't think that would be necessary, as the attributes can be often SoC specific (e.g. not all devices define the same domains and partitions). The current change is somewhat similar to what is already done for other types of attributes like https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/base/zephyr%2Cmemory-common.yaml#L22-L27

Reply:
Updated, thanks.

Reply:
well, the reference to the file with the macros is something, I guess...

Reply:
> I don't think that would be necessary, as the attributes can be often SoC specific (e.g. not all devices define the same domains and partitions). The proposed change on this pr is somewhat similar to what is already done for other types of attributes like https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/base/zephyr%2Cmemory-common.yaml#L22-L27

something is necessary, user when looking at binding documentation should not have to guess